prow/plugins/updateconfig/updateconfig.go

```diff
@@ -315,3 +321,16 @@ func handle(gc githubClient, kc corev1.ConfigMapsGetter, defaultNamespace string
 	}
 	return nil
 }
+
+// GetConfigMapClient returns a configMap interface according to the given cluster and namespace
+func GetConfigMapClient(kc corev1.ConfigMapsGetter, namespace string, buildClusterCoreV1Clients map[string]corev1.CoreV1Interface, cluster string) (corev1.ConfigMapInterface, error) {
```

Comment:
I tried to write unit tests for this function.
But `gomock` seems very complicated.

I tested `value, ok := map[key]` with map=nil ... works as @stevekuznetsov said: `!ok` instead of `nil` pointer error.